src/app/users/store/users.actions.ts

```diff
@@ -3,16 +3,21 @@ import { IUser } from '../user.interface';
 
 const begin = createAction('[Users] Get All - Begin');
 const success = createAction(
-  '[Users] Get All - Success',
+  '[Users Effect] Get All - Success',
   props<{ users: IUser[] }>()
 );
-const failure = createAction(
-  '[Users] Get All - Failure',
-  props<{ error: any }>()
+
+//delete user
+const deleteUserBegin = createAction(
+  '[Users] Delete - Begin',
```

Comment:
Here you made it right,  Always check that you should be consistent on naming conventions.

Reply:
done!!